client/material/tooltip.tsx

```diff
@@ -188,10 +208,23 @@ export function Tooltip({
         }
       }
     } else {
-      setOpen(false)
-      return () => {}
+      if (interactive) {
+        let timeout: ReturnType<typeof setTimeout> | undefined = setTimeout(() => {
```

Comment:
I feel like it would probably make more sense to have padding around the tooltip that would keep it open if you were still in the vicinity, rather than delaying the close (I assume that's what this is for anyway, can't really see another purpose for it). Delay on closing can make the interface feel laggy/unresponsive